Tests/DatadogTests/Datadog/RUMMonitorTests.swift

```diff
@@ -1387,6 +1387,34 @@ class RUMMonitorTests: XCTestCase {
         XCTAssertEqual(transformedCommand.time, Date(timeIntervalSince1970: 1)) // 1 in seconds
     }
 
+    // MARK: - Cross-platform performance metrics
+
+    func testWhenViewIsStarted_performanceMetricsAreSent() throws {
+        let rum: RUMFeature = .mockByRecordingRUMEventMatchers()
+        core.register(feature: rum)
+
+        let monitor = try createTestableRUMMonitor()
+
+        monitor.startView(viewController: mockView)
+        monitor.updatePerformanceMetric(metric: PerformanceMetric.JS_REFRESH_RATE, value: 40.0)
+        monitor.updatePerformanceMetric(metric: PerformanceMetric.JS_REFRESH_RATE, value: 20.0)
+        monitor.updatePerformanceMetric(metric: PerformanceMetric.FLUTTER_BUILD_TIME, value: 32.0)
+        monitor.updatePerformanceMetric(metric: PerformanceMetric.FLUTTER_RASTER_TIME, value: 42.0)
```

Comment:
Same here, you don't need to qualify the enums in Swift. These can be simplified to:
```swift
monitor.updatePerformanceMetric(metric: .jsRefreshRate, value: 40.0)
```